### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by Dog2puppy**

### CODE REVIEW

File: responsive-domain-search/responsive-domain-search.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ document.addEventListener("DOMContentLoaded", function() {
     
     /* 
      *  Created at 10 July 2018 by PlanetCloud.
-     *  Last modified at 02 November 2018 by PlanetCloud.
+     *  Last modified at 16 March 2019 by Dog2puppy.
```
Comment (PCTipsGR) [inline responsive-domain-search/responsive-domain-search.js]: Please keep the changelog here, you could put it like this:
```
    /* 
     *  Created at 10 July 2018 by PlanetCloud.
     *  Modified at 02 November 2018 by PlanetCloud.
     *  Last modified at 16 March 2019 by Dog2puppy.
     *  DO NOT REMOVE CREDITS!
     *  Created for: VPT (VistaPanel Team).
     */
```

Comment (Dog2puppy) [reply responsive-domain-search/responsive-domain-search.js]: Fixed.

**Comment by Dog2puppy** [comment]
Also, why do you manually make .min.js files when jsDelivr auto generates them?

**Comment by PCTipsGR** [comment]
> Also, why do you manually make .min.js files when jsDelivr auto generates them?

At the time of the project creation, we used RawGit, which did not auto-generate these files AFAIK. That is why we had created them manually.
We will consider removing them, if that is the case, thanks for the heads-up.

File: responsive-domain-search/responsive-domain-search.js (hunk 1 above)
Comment (PCTipsGR) [reply responsive-domain-search/responsive-domain-search.js]: Pardon? How is it fixed? Where did you change it?

**Comment by Dog2puppy** [comment]
Documentation said it was supposed to be affCode. Can be changed back though if needed.

**Comment by PCTipsGR** [comment]
> I don't think the var name here should be changed as because PlanetCloud change var name here to save some bytes and bytes matter in speed of site.

> Documentation said it was supposed to be affCode. Can be changed back though if needed.

@mahofficial I agree with @Dog2puppy on this, either that or the documentation needed a fix, which is exactly why we need the review from @PlanetGamingGG.

**Comment by mahofficial** [comment]
> > I don't think the var name here should be changed as because PlanetCloud change var name here to save some bytes and bytes matter in speed of site.
> 
> > Documentation said it was supposed to be affCode. Can be changed back though if needed.
> 
> @mahofficial I agree with @Dog2puppy on this, either that or the documentation needed a fix, which is exactly why we need the review from @PlanetGamingGG.

Yeah! Documentation needed a fix.

**Comment by PCTipsGR** [comment]
Well, since the doc was outdated, the variable got updated, and I think it is better because "affcode" explains the purpose of the variable better to someone new to the code.

File: responsive-domain-search/responsive-domain-search.js (hunk 1 above)
Comment (PCTipsGR) [reply responsive-domain-search/responsive-domain-search.js]: Fixed now.

**Comment by mahofficial** [comment]
> Well, since the doc was outdated, the variable got updated, and I think it is better because "affcode" explains the purpose of the variable better to someone new to the code.

Well planetcloud changed the var `affcode` to `a` to save some bytes which is considered as better for the speed of website. Let's leave merge on him. lol

File: responsive-domain-search/responsive-domain-search.js (hunk 1 above)
Comment (Dog2puppy) [reply responsive-domain-search/responsive-domain-search.js]: GitHub must've not saved it then

**Pull request merged**